homeassistant/components/cloud/http_api.py

```diff
@@ -191,6 +192,9 @@ class GoogleActionsSyncView(HomeAssistantView):
     @_handle_cloud_errors
     async def post(self, request: web.Request) -> web.Response:
         """Trigger a Google Actions sync."""
+        if not request["hass_user"].is_admin:
+            raise Unauthorized()
+
```

Comment:
Why don't create a decorator for it?
Like `@require_admin` for websockets but for `HomeAssistantView`.
Similar to `@_handle_cloud_errors`

The best would it be to place it in `homeassistant.components.http`
In a followup PR we could use the new decorator on ~20 other places (searched for `request["hass_user"].is_admin`)

Reply:
makes sense 👍

Reply:
I added it as a generic decorator in the HTTP integration, so it can be re-used in all other places.